daemon/volumes.go

```diff
@@ -27,9 +28,12 @@ type mounts []container.Mount
 
 // volumeToAPIType converts a volume.Volume to the type used by the Engine API
 func volumeToAPIType(v volume.Volume) *types.Volume {
+	createdAt, _ := v.CreatedAt()
```

Comment:
Looking at the code, we always return a `time.Time` here (if there is an error in the `windows` code path it will return `time.Time{}`) so I think it's safe to ignore the error here 👼